packages/application/src/app.ts

```diff
@@ -24,12 +25,14 @@ export type QuetzFrontEndPlugin<T> = IPlugin<QuetzFrontEnd, T>;
  * App is the main application class. It is instantiated once and shared.
  */
 export class App extends Application<Shell> {
+  private _info: JupyterLab.IInfo = JupyterLab.defaultInfo;
+
   /**
    * Construct a new App object.
    *
    * @param options The instantiation options for an application.
    */
-  constructor(options: App.IOptions = {}) {
+  constructor(options: App.IOptions = { shell: new Shell() }) {
```

Comment:
This is unneeded as L38 gets us covered.

```suggestion
  constructor(options: App.IOptions = {}) {
```